www/js/controllers.js

```diff
@@ -8,18 +8,20 @@ angular.module('emission.controllers', ['emission.splash.updatecheck',
                                         'emission.survey.launch',
                                         'emission.stats.clientstats',
                                         'emission.incident.posttrip.prompt',
-                                        'customURLScheme'])
+                                        'customURLScheme',
+                                        'emission.splash.ionicNotification'])
```

Comment:
did you actually test this? I get the following error when I try to deploy after merging.

```
Uncaught Error: [$injector:modulerr] Failed to instantiate module emission due to:
Error: [$injector:modulerr] Failed to instantiate module emission.controllers due to:
Error: [$injector:modulerr] Failed to instantiate module emission.splash.ionicNotification due to:
Error: [$injector:nomod] Module 'emission.splash.ionicNotification' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.
http://errors.angularjs.org/1.5.3/$injector/nomod?p0=emission.splash.ionicNotification
```

Please indicate how you have tested the code before you send the PR...